components/collective-page/CollectiveNotificationBar.js

```diff
@@ -98,6 +98,9 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host, LoggedInUser, refetch) => {
+  const numberOfAdmins = [CollectiveType.EVENT, CollectiveType.PROJECT].includes(collective.type)
+    ? collective?.parentCollective?.coreContributors?.filter(c => c.isAdmin)?.length
```

Comment:
Now that children collectives can have their own admins, we still need to sum this with `collective.admins.lenght`.

Reply:
Thanks. Corrected. 👍🏽